arm-network/2016-09-01/swagger/network.json

```diff
@@ -8041,10 +8120,14 @@
           "description": "Gets or sets the certificate public data"
         },
         "provisioningState": {
+          "readOnly": true,
           "type": "string",
           "description": "Gets provisioning state of the VPN client root certificate resource Updating/Deleting/Failed"
         }
       },
+      "required": [
+        "publicCertData"
+      ],
       "description": "Properties of SSL certificates of application gateway"
     },
     "VpnClientRootCertificate": {
```

Comment:
same reason.
"properties" property of  "VpnClientRootCertificate" should also be required.